lib/helpers/languages_subtitle.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:collection/collection.dart';
+
+import 'languages.dart';
+
+const noneLanguageCode = 'none';
+
+const subtitleLanguages = [
+  LanguageInfo(
+    code: noneLanguageCode,
+    nativeName: 'None',
+  ),
+  ...languages
+];
+
+String? getSubtitleLanguageName(String? languageCode) {
```

Comment:
We can skip this file by:
- Using null instead of a special code
- Make OptionList accept a new argument, e.g. `showNoneOption` which is optional and false by default